src/pages/leeszaal/[slug].tsx

```diff
@@ -114,6 +115,18 @@ const ReadingRoomPage: NextPage = () => {
 		}
 	};
 
+	const onNewKeyWord = (newKeyWord: string) => {
+		setQuery({ search: (query.search ?? []).concat(newKeyWord) });
+	};
+
+	const onResetFilters = () => {
+		setQuery({ search: undefined });
+	};
+
+	const onRemoveFilter = (newValue: SearchBarValue<true>) => {
+		setQuery({ search: newValue.map((tag) => tag.value as string) });
+	};
```

Comment:
Is het niet eenvoudiger (& immutable) om steeds een array output te ontvangen van de `SearchBar` en `search` simpelweg te overschrijven in plaats van string concat & replace? Via bv. `activeFilters`?

Reply:
De query is in dit geval de source of truth dus de filters gaan steeds aangepast worden aan wat er uit de query komt. Er moet toch zwz een mapping komen op de values die we terugkrijgen van de SearchBar.